CourseNetworkAPI/CourseNetwork.cs

```diff
@@ -0,0 +1,205 @@
+﻿#region Author Information
+/*
+ * Andrue Cashman
+ * University of Washington Bothell
+ * Undergraduate Research Winter 2018
+ * Advisor: Dr. Erika Parsons
+ * Project: Virtual Student Advisor
+*/
+#endregion
+
+#region CourseNetwork Information
+/*
+ * CourseNetwork
+ * Purpose: To build a network of courses and their respective prerequisites to be used in the 
+ *          Virtual Student Advisor Project in assisting students and/or advisors in scheduling
+ *          courses.
+ * 
+ * Assumptions: All courses and prerequisiste information are correct in that they are saved as 
+ *              json string files.
+ *                  -SQL queries for correct JSON structure based on the VSA database
+ *                      -AllCourses: select CourseID from Course for JSON AUTO;
+ *                      -PrereqNetwork: Select CourseID, GroupID, PrerequisiteID from Prerequisite for JSON AUTO
+ *              All information from the database is correct.
+ *              CourseNode is in the structure of int CourseID, int GroupID, int PrerequisiteID, List<CourseNode> prereqs
+*/
+#endregion
+
+#region Libraries and Namespaces
+using System;
+using System.Collections.Generic;
+
+using Newtonsoft.Json; //IMPORTANT: Used for translating the JSON strings into a usable format
+using Newtonsoft.Json.Linq;
+using System.Data.SqlClient;
+using System.IO;
+using System.Text;
+using System.Diagnostics;
+#endregion
+
+namespace CourseNetworkAPI
```

Comment:
Move this to APICore so it can be tested. Idea = Api is super lightweight and APICore contains the main logic and has UTs (someday!)